### PR TITLE
[TASK] Require TYPO3 >= 6.2

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

File: tests/widgets/class.tx_mkforms_tests_widgets_fluidviewhelper_testcase.php [hunk 1]
```diff
@@ -1,221 +0,0 @@
-<?php
```
Comment (hannesbochmann) [inline tests/widgets/class.tx_mkforms_tests_widgets_fluidviewhelper_testcase.php]: this class shouldn't be dropped

File: tests/widgets/class.tx_mkforms_tests_widgets_text_Main_testcase.php [hunk 2]
```diff
@@ -1,139 +0,0 @@
-<?php
```
Comment (hannesbochmann) [inline tests/widgets/class.tx_mkforms_tests_widgets_text_Main_testcase.php]: this class shouldn't be dropped

File: tests/widgets/class.tx_mkforms_tests_widgets_fluidviewhelper_testcase.php (hunk 1 above)
Comment (oliverklee) [reply tests/widgets/class.tx_mkforms_tests_widgets_fluidviewhelper_testcase.php]: Oops. Fixed.

File: tests/widgets/class.tx_mkforms_tests_widgets_text_Main_testcase.php (hunk 2 above)
Comment (oliverklee) [reply tests/widgets/class.tx_mkforms_tests_widgets_text_Main_testcase.php]: Fixed and repushed.

**Pull request merged**